src/lib/viewers/controls/color-picker/ColorPickerToggle.scss

```diff
@@ -0,0 +1,21 @@
+@import '~box-ui-elements/es/styles/variables';
+@import '../styles';
+
+.bp-ColorPickerToggle {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+}
+
+.bp-ColorPickerToggle-button {
+    @include bp-ControlButton;
+
+    svg {
+        background-color: #0061d5;
```

Comment:
I'll remove this line once we have the AnnotationModule business logic implemented.